src/platform/user/authentication/components/OrganicAdoptionExperimentModal.jsx

```diff
@@ -9,6 +9,17 @@ import { isAuthenticatedWithOAuth } from '../selectors';
 const OrganicAdoptionExperimentModal = ({ visible = false, onClose }) => {
   const useOAuth = useSelector(isAuthenticatedWithOAuth);
 
+  useEffect(
+    () => {
+      recordEvent({
+        event: `organic-adoption-experiment-modal-${
+          visible ? 'opened' : 'closed'
+        }`,
+      });
+    },
+    [visible],
```

Comment:
This probably needs to be changed. This will fire  the recordEvent `closed` on load instead of on if it was actually closed by the user. There were some Google Analytics issue on how this was working with the SignInModal component was working last week.

Reply:
I only did it this way because my other events are not ending up in GA dashboard and I was wondering if they need to happen in the hook. I'll remove for now.

Reply:
I also saw that in the user-nav/Main.jsx that the visibility is `null` instead of false for the state, I wasn't sure if it was supposed to be like that or get updated to be false.